src/components/client/campaigns/CampaignDetails.tsx

```diff
@@ -138,9 +138,6 @@ export default function CampaignDetails({ campaign }: Props) {
         showExpensesLink={(expensesList && expensesList?.length > 0) || canEditCampaign}
       />
       <Grid container spacing={8}>
-        {subscribeIsOpen && (
-          <RenderCampaignSubscribeModal setOpen={setSubscribeOpen} campaign={campaign} />
```

Comment:
Would you please confirm the removal of this fragment, doesnt cause regression in opening the subscription modal when user clicks on the `Subscribe for email notifications for the campaign.`